roles/kubernetes/sas-iac-buildinfo/tasks/main.yaml

```diff
@@ -13,39 +13,40 @@
 #       is handled externally and therefore will not
 #       have this code applied.
 #
-- block:
-  - name: Register Git HASH information
-    ansible.builtin.shell: |
-      cd "{{ k8s_tool_base }}"
-      "{{ k8s_tool_base }}/files/tools/iac_git_info.sh"
-    register: git_hash
-    tags:
-      - install
-      - update
+- name: Setting up sas-iac-buildinfo
+  when: deployment_type == 'vsphere'
```

Comment:
Tags and conditionals after blocks

Reply:
The order was updated to follow ansible conventions based off this rule: 
https://ansible.readthedocs.io/projects/lint/rules/key-order/